Explain timing separation of privacy pass contexts (#55)
intarchboard/draft-iab-privacy-partitioning: draft-iab-privacy-partitioning.md

~~~
{: #diagram-privacypass title="Diagram of contexts in Privacy Pass"}

Since the redemption context and issuance context are separate connections
that involve separate entities, they can also be further decoupled by

> running those parts of the protocols at very different times. Clients can

Reviewer: ```suggestion
running those parts of the protocols at different times. Clients can
```

fetch tokens through the issuance context early, and cache the tokens

> to later use in redemption contexts. This aids in partioning identifiers

Reviewer: ```suggestion
to later use in redemption contexts. This can aid in partitioning identifiers
```

Reviewer: "Can" since separation over time doesn't help in the extreme case where there is just one issuance and redemption.

and data.

> {{PRIVACYPASS}} describes different models for which entities operate

Reviewer: ```suggestion
{{PRIVACYPASS}} describes different deployment models for which entities operate
```
